flycheck.el

```diff
@@ -7722,8 +7722,8 @@ See URL `https://github.com/commercialhaskell/stack'."
             source)
   :error-patterns
   ((warning line-start (file-name) ":" line ":" column ":"
-            (or " " "\n    ") "Warning:"
-            (optional " " "[" (id (one-or-more not-newline)) "]")
+            (or " " "\n    ") (in "Ww") "arning:"
+            (optional " " "[" (id (one-or-more not-newline)) "] ?")
```

Comment:
you're right ... not sure how that ever caught anything. maybe i'll get rid of that change and just do the case change.